### ISSUE TITLE
Failure using Backup Ansible role

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#2019]
## Description

Backups fail using "Backup and restore" on 0.4

## Steps to Reproduce

1. Install 0.4 from test debs and upgrade admin workstation
2. `source .venv/bin/activate`
3. `cd install_files/ansible-base`
4. `ansible-playbook -t backup securedrop-prod.yml -e perform_backup=true`

## Expected Behavior

The backup completes successfully and a backup file `sd-backup*` should appear in the current working directory 

## Actual Behavior

Backup fails:

```
TASK [backup : fetch the backup file] ******************************************
fatal: [app]: FAILED! => {"failed": true, "msg": "the field 'args' has an invalid value, which appears to include a variable that is undefined. The error was: 'dict object' has no attribute 'stdout'\n\nThe error appears to have been in '/home/amnesia/Persistent/securedrop/install_files/ansible-base/roles/backup/tasks/backup.yml': line 17, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: fetch the backup file\n  ^ here\n"}
	to retry, use: --limit @/home/amnesia/Persistent/securedrop/install_files/ansible-base/securedrop-prod.retry
```

**Comment by conorsch** [comment]
Will take a look. Worth mentioning that a permanent solution here should include:

1. a dedicated playbook just for the backup action (#1742)
2. an entry point in the new admin script, e.g. `./securedrop-admin backup`

But since we are already in QA, I will see if a narrower solution will suffice.

**Comment by redshiftzero** [comment]
Totally, `securedrop-admin backup` and `securedrop-admin restore <restore_file>` options would be excellent - added followup issue #2023 for tracking

**Comment by conorsch** [comment]
The problem appears to be the `async` parameter on the `run the backup script` task, which effectively causes the task to be backgrounded while the subsequent tasks are executed serially. That's no good, because we need the registered result of the "backup_filename" var that's returned from the `run the backup script` task. 

The original intent of the `async` inclusion was to prevent timeouts from long-running tasks. We can still achieve that, but we'll need to explicitly wait on the backgrounded task. Will cobble something together and get up a PR for review.

**Comment by conorsch** [comment]
Essentially a one-line change: we need an explicit `poll:` parameter on the async task. PR incoming.

**Comment by redshiftzero** [comment]
Fixed in #2026
